aprotos/07_CSS2/hw_5flex.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+   <meta charset="utf-8">
+   <title>Flex1</title>
+   <style>
+      .containter {
+         display: flex;
+         align-items: flex-start;
+      }
+      .navigation {
+         display: flex;
+         flex-direction: column;
+         flex: 0 0 15em;
+         background-color: purple;
+      }
+      .content {
+         margin-bottom: .5em;
+         background-color: green;
+      }
+      .lowwer {
+         background-color: grey;
+      }
+      .send {
+         float: right;
+         padding:7px;
+         }
+   </style>
+</head>
+<body>
+   <div class="containter">
+      <div class="navigation">
+      <ul>
+            <li>Главная</li>
+            <li>Каталог</li>
+            <li>Каталог</li>
+            <li>О нас</li>
+            <li>Контакты</li>
+            <li>Информация</li>
+            <li>Подробнее</li>
+            <li>Обратная связь</li>
+      </ul>
+      </div>
+         <div>
```

Comment:
here should be class which provide flex: ? ? ?, you need to put a rule for this "column"